Packs/Core/Playbooks/playbook-Uncommon_creation_or_access_operation_of_sensitive_shadow_copy_by_a_high_risk_process.yml

```diff
@@ -0,0 +1,971 @@
+      name: Check if CGO is signed
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "17"
+      CGO-Signed:
+      - "33"
+    separatecontext: false
+    conditions:
+    - label: CGO-Signed
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: alert.osparentsignature
+            iscontext: true
+          right:
+            value:
+              simple: SIGNATURE_SIGNED
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 705,
+          "y": -105
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: f0154772-d5ec-42cc-8053-0cf4f57da493
+    type: condition
+    task:
+      id: f0154772-d5ec-42cc-8053-0cf4f57da493
+      version: -1
+      name: Check CGO image name
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "13"
+      cmd.exe|rundll32.exe:
+      - "10"
+      powershell.exe:
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: powershell.exe
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: powershell.exe
+          ignorecase: true
+    - label: cmd.exe|rundll32.exe
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: cmd.exe
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: alert.cgoname
+            iscontext: true
+          right:
+            value:
+              simple: rundll32.exe
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 110,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 3538379b-1a77-4f01-8688-2fe1caa4c8ec
+    type: condition
+    task:
+      id: 3538379b-1a77-4f01-8688-2fe1caa4c8ec
+      version: -1
+      name: Check actor_process_image_name
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "10"
+      (powershell.exe | ntdsutil.exe | esentutl.exe | cmd.exe:
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: (powershell.exe | ntdsutil.exe | esentutl.exe | cmd.exe
```

Comment:
in one task you made spaces before and after the "|", in the other one no spaces, please be consistent. 
I think the correct grammar is without spaces

Reply:
what about that?